scalableminds/webknossos
# Dataset upload: Fix special case in common prefix handling

When uploading a single file inside a zipfile, the common prefix handling ran into a bug, breaking the whole upload. This is fixed here.

### Steps to test:
- set jobsEnabled = true (no need to set up worker)
- create zip file with single tif inside
- upload that as dataset, no exception should be logged (only that the worker can’t be reached, if not set up)

### Issues:
- fixes #5529 

------
